tests/integrationtest/t/planner/core/integration.test

```diff
@@ -234,31 +234,6 @@ INSERT INTO t0(c0) VALUES (3);
 SELECT /*+ MERGE_JOIN(t1, t0, v0)*/v0.c2, t1.c0 FROM v0,  t0 CROSS JOIN t1 ORDER BY -v0.c1;
 
 
-# TestInvisibleIndex
```

Comment:
Unstable and duplicated with `bindinfo/test/TestInvisibleIndex`, just remove it.